compile.js

```diff
@@ -89,70 +96,79 @@
    * Compile a client-side library based on the flickr-api-object code.
    */
   new APIBuilder(flickrOptions, Utils, function(err, flickr) {
-    if(err) {
+    if (err) {
       console.error(err);
       process.exit(1);
     }
 
     delete flickr.options;
-    buffer.write("(function() {");
+    // buffer.write("(function() {");
+    buffer.write(wrapper[0]);
 
     // library-specific Utils
     buffer.write(" var Utils = {};");
     writeTree(require("./browser/Utils.js"), "Utils", buffer.write);
-    buffer.write(" Utils.errors = " + JSON.stringify(Utils.getCallErrors(),false, 4) + ";");
+    buffer.write(" Utils.errors = " + JSON.stringify(Utils.getCallErrors(), false, 4) + ";");
 
     // Flickr object definition
     buffer.write(" var Flickr = " + (function(flickrOptions) {
-  this.bindOptions(flickrOptions);
-}).toString() + ";");
+      this.bindOptions(flickrOptions);
+    }).toString() + ";");
     buffer.write(" Flickr.prototype = {};");
 
     // Prototype from methods
     setupMethods(flickr);
 
-    if (process.argv.indexOf("dev") === -1) { stripDev(methods); }
-    buffer.write(" Flickr.methods = " + JSON.stringify(methods,false,1) + ";");
+    if (process.argv.indexOf("dev") === -1) {
+      stripDev(methods);
+    }
+    buffer.write(" Flickr.methods = " + JSON.stringify(methods, false, 1) + ";");
     var fn = (function() {
-  Object.keys(Flickr.methods).forEach(function(method) {
-    var level = method.split(".").slice(1);
-    var e = Flickr.prototype, key;
-    while(level.length > 1) {
-      key = level.splice(0,1)[0];
-      if(!e[key]) { e[key] = {}; }
-      e = e[key];
+      Object.keys(Flickr.methods).forEach(function(method) {
```

Comment:
same here, all this spacing needs to be restored because the functions get `toString()`ed and then written to file, so the spaces used here genuinely matter for the purpose of result spaces in the browser `.js` library files.